operator/pkg/ingress/envoy_config.go

```diff
@@ -627,8 +637,8 @@ func getRedirectRouteConfigurationResource(ingress *slim_networkingv1.Ingress) (
 // Currently, only TPC keep-alive related options are specified.
 //
 // Related references:
-//  - https://man7.org/linux/man-pages/man7/tcp.7.html
-//  - https://github.com/envoyproxy/envoy/issues/3634
+//   - https://man7.org/linux/man-pages/man7/tcp.7.html
+//   - https://github.com/envoyproxy/envoy/issues/3634
```

Comment:
What happened here? :thinking:

Reply:
probably it's auto format that I have configured in IDE. Golang recently added format for godocs as well :(